### DISCUSSION TITLE
Github Actions should provide a github signed JWT access token

### DISCUSSION THREAD

**Discussion by wparad** [community/community#4742] [Actions] [answered]
The current token provided by GitHubActions doesn't cut it to integrate and log in to other systems. For instance AWS requires a registered OAuth provider configuration, and GitHub Actions current tokens don't work with that. It should be easy for github actions to inject in an environment variable with a signed JWT that external systems can verify:
* Github issuer
* repo generating token ([github/feedback](https://github.com/github/feedback) in this case)
* type of event
* ref source
* full project information ([github](https://github.com/github))

**Comment by ethomson** [comment, marked as answer]
Hi @wparad - thanks for the feedback.  We now support [OpenID Connect](https://github.blog/changelog/2021-10-27-github-actions-secure-cloud-deployments-with-openid-connect/) which we hope will work for you!
